Storage/Bookmarks/CachingItemSource.swift

```diff
@@ -174,6 +174,10 @@ open class CachingBufferItemSource: BufferItemSource {
         return self.prefetchBufferItemsWithGUIDs(guids) >>> { self.cache.takingGUIDs(guids) }
     }
 
+    public func getBufferChildrenGUIDsForParent(_ guid: GUID) -> Deferred<Maybe<[GUID]>> {
+        return deferMaybe(DatabaseError(description: "Not implemented"))
```

Comment:
Might as well make this `self.source.getBufferChildrenGUIDsForParent(guid)`. Same amount of code, but it actually does something!